registry.json

```diff
@@ -1918,5 +1918,29 @@
       "rpc": "https://smartbch.fountainhead.cash/mainnet",
       "documentation": "https://github.com/smartbch/docs/blob/main/developers-guide/jsonrpc.md"
     }
+  },
+  {
+    "id": "kcc",
+    "name": "KuCoin Community Chain",
+    "coinId": 10000321,
```

Comment:
I have an issue with coinId: we use the scheme of 10000000 + Id if two chains have the same coin (e.g. 714 for BNB on BC, 10000714 for BNB on BSC).
In this case 321 is not taken, so no justification for using 8-digit code.
Also, we always require the coinId to be listed in
https://github.com/satoshilabs/slips/blob/master/slip-0044.md
I don't see it there, can you add it, to make sure Id is not used by other chain?
321 is listed for Rapture RAP.

Reply:
OK, discussed this over.  Primary KCS is an ERC-20 token, so it does not have a Slip44 ID.
For KCC, coinID should be 10000000 + EthereumChainID, so 10000321 is fine.